src/steps/when.js

```diff
@@ -97,5 +97,37 @@ module.exports = function () {
             }
         });
 
+        When(_r('I switch to first tab$'), async function async () {
+            /**
+             * Open last tab
+             * @param {String} err - Any error message
```

Comment:
This function does not have any incoming parameters, so you do not need to describe any `@param`

Reply:
Implemented